pkg/config/config.go

```diff
@@ -117,6 +124,10 @@ type NsxConfig struct {
 	EnvoyHost                 string   `ini:"envoy_host"`
 	EnvoyPort                 int      `ini:"envoy_port"`
 	LicenseValidationInterval int      `ini:"license_validation_interval"`
+	UseAVILB                  bool     `ini:"use_avi_lb"`
+	UseNativeLoadBalancer     *bool    `ini:"use_native_loadbalancer"`
+	RelaxScaleValidaion       bool     `ini:"relax_scale_validation"`
+	ServiceSize               string   `ini:"service_size"`
```

Comment:
Why it is called "ServiceSize"? "NsxLbSize"?

Actually, could you explain how these parameters are populated? I wonder multiple things not consistent with our naming convention:

1. Why we use "Nsx", not "NSX".
2. Why "UseNativeLoadBalancer', not "UseNSXLoadBalancer".
3. Why "RelaxScaleValidation", not "RelaxNSXLBScaleValidation"
4. Why we use all upper cases for values like "SMALL".

Reply:
All the options in this file is already used by NCP. We just import it into nsx-operator.